packages/@sanity/cli/templates/shopify/components/media/ColorTheme.tsx

```diff
@@ -0,0 +1,27 @@
+type Props = {
+  background?: string
+  text?: string
+}
+
+const ColorThemePreview = (props: Props) => {
+  const {background, text} = props
+
+  return (
+    <div
+      style={{
```

Comment:
Prefer styled component instead of inline styling

```ts
import styled, {css} from 'styled-components'

const StyledSpan = styled.span(({$background}) => {
  const bg = $background || 'white'

  return css`
    align-items: center;
    background-color: ${bg};
    border-radius: inherit;
    display: flex;
    height: 100%;
    justify-content: center;
    overflow: hidden;
    width: 100%;
  `
})

```